src/fixtures/item.ts

```diff
@@ -36,32 +34,38 @@ const fields = {
 const article: Item = {
     design: Design.Article,
     ...fields,
+    mainMedia: { kind: MainMediaKind.Image, image: new None() }
```

Comment:
Are you able to put this line in the `fields` object above to avoid repeating it each time?